VideoLocker/src/main/java/org/edx/mobile/http/Api.java

```diff
@@ -1266,4 +1270,52 @@ public Boolean enrollInACourse(String courseId, boolean email_opt_in) throws Exc
         return false;
     }
 
+
+    /**
+     * Returns course structure object from the given URL.
+     * @param courseId
+     * @param preferCache
+     * @return
+     * @throws Exception
+     */
+    public String getCourseStructure(String courseId, boolean preferCache) throws Exception {
```

Comment:
@aleffert  yeah. we don't use it for now. just make it consistent with previous api.


ideally there should be another layer of code, service,  the service will delegate the call to database/server api.. ,    and  the logic of  client side cache should happen at service layer.